.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,3 +9,10 @@ Make sure that the following steps are done before merging:
     - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
     - [ ] Add an entry to the CHANGELOG.
   - [ ] Have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?
+
+If changes are made as part of the base installation, then you need to issue a pull requests to all the basic branches:
+- [ ] ficus-rg
+- [ ] gingko-rg
+- [ ] hawthorn-rg
+- [ ] development gingko-rg
```

Comment:
what's this branch is for? what is the difference with ginkgo-rg?

Reply:
this is the mention of RG's private repositories